CONTRIBUTING.md

```diff
@@ -0,0 +1,338 @@
+
+##### tags
+
+Required field, tag, which is used to categorize the blog. Each post has and can only have one tag . If more than one tag is suitable, please choose the most suitable one. The available tags and explanations are as follows.
+
+- Community: community related, e.g. "How to contribute to an open source project without writing code?"
+- Events: event-related, for example: online live, preview, offline meeting, meetup, etc.
+- Interview: Interviews, e.g., Dr. Yang Li interview, Summer of Programming interview.
+- Practical Case: Best practices, easily confused with Technology. The factors that determine whether an article is a Technology or a Practical Case are: the content of the article and the subject of the description. For example, if the article is about "Running Apache APISIX on xxx platform", then it is a Practical Case; for example, if the article is about technical stuff, "Apache APISIX vs. ", then it is Technology.
+- Release: Release notes, this is better understood. It should be noted that the release notes inside the blog are embellished, while the release notes inside the release are pure dry goods.
+- Security: Security vulnerability notification and methods to bypass security vulnerabilities, currently there are only two articles, very good to identify, generally have CVE-xxxxxxx is it.
+- Technology: Technical articles, easily confused with Practical Case. The factors that determine whether an article is a Technology or a Practical Case are: the content of the article and the subject of the description. For example, if the article is about "Running Apache APISIX on xxx platform", then it is a Practical Case; for example, if the article is mainly about technical stuff, "Apache APISIX vs. ", then it is Technology.
+- User Case: User Case, this is also very recognizable, so I won't talk about it.
+
+#### How to get the authorImageURL
+
+1. Open Chrome
+2. Enter the author's GitHub address to access the author's GitHub home page
+3. Fn + F12 or just press F12
+4. Find the link that corresponds to the author's avatar
+5. Copy and paste it into the authorImageURL field
+![How to get the authorImageURL](https://static.apiseven.com/202108/1637920593469-488c0905-c81f-40f9-90b5-19ffafc54147.png)
+
+#### truncate and overview
+
+```
+> overview
+
+<!--truncate-->
+```
+
+An overview is entered starting with > and a space followed by an overview. The overview can be your own summary of the article, or it can be the first paragraph of the article with a summary description. When you are done, type `<! --truncate-->`. `<! --truncate--->` serves to display the `<! --truncate--->` above the blog overview, or if there is no overview, nothing will be displayed; if there is an overview but no `<! --truncate--->`, it will show the whole thing.
```

Comment:
```suggestion
An overview is entered starting with `>` and a space followed by texts. The overview can be your own summary of the article, or it can be the first paragraph of the article with a summary description. When you are done, type `<! --truncate-->`. `<! --truncate--->` serves to display the `<! --truncate--->` above the blog overview, or if there is no overview, nothing will be displayed; if there is an overview but no `<! --truncate--->`, it will show the whole thing.
```